.github/workflows/scripts/translation_check.py

```
@@ -0,0 +1,295 @@
+"""Validates i18n translation tags against locale JSON files."""
```

Ruff (0.14.10)

The file is executable but no shebang is present

(EXE002)